docs/cli/custom-objects-exporter.md

```diff
@@ -0,0 +1,87 @@
+# Custom Objects Exporter
+
+A package that helps with exporting [commercetools custom objects](https://docs.commercetools.com/http-api-projects-custom-objects.html) in `JSON` format from the [commercetools platform](https://docs.commercetools.com/).
+
+## Configuration
+
+The constructor accepts two arguments:
+
+* A required object containing the following values:
+  * `apiConfig` (Object): `AuthMiddleware` options for authentication on the commercetools platform. (Required. See [here](https://commercetools.github.io/nodejs/sdk/api/sdkMiddlewareAuth.html#named-arguments-options))
+  * `accessToken` (String): Access token to be used to authenticate requests to API. Requires scope of [`view_products`], [`view_orders`], [`view_customers`]
+  * `predicate` (String): Query string specifying (where) predicate. More info on predicates [here](https://docs.commercetools.com/http-api.html#predicates) (Optional)
+* An optional logger object having four functions (`info`, `warn`, `error` and `verbose`)
```

Comment:
I think it'll make more sense to use `debug` as the fourth log level instead of `verbose`. That's also what you pass in [here](https://github.com/commercetools/nodejs/pull/543/files#diff-df95a275d2c8c414f98590424a9d9b75R76)